recipe/meta.yaml

```diff
@@ -24,7 +24,7 @@ requirements:
   run:
     - python
     - pickleshare
-    - python-simplegeneric >0.8
+    - simplegeneric >0.8
```

Comment:
Ping @pelson not sure if this issue was "closed" yet, but if you merge this I will rename the [feedstock](https://github.com/conda-forge/python-simplegeneric-feedstock).